plugins/services/src/js/reducers/serviceForm/JSONReducers/Containers.js

```diff
@@ -392,7 +392,9 @@ module.exports = {
             newState
           );
 
-          newState.push(Object.assign({}, DEFAULT_POD_CONTAINER, { name }));
+          newState.push(
+            Object.assign({}, DEFAULT_POD_CONTAINER, value, { name })
```

Comment:
Actually, we want to overwrite the name with the name in the value if there is one. Although the parser will generate a transaction for the name anyway. But I would prefer it to be right. 
```JS
Object.assign({}, DEFAULT_POD_CONTAINER, { name }, value)
```

Reply:
🤦‍♂️